vendor/vendor.json

```diff
@@ -48,10 +48,12 @@
 			"revisionTime": "2017-09-25T15:48:32Z"
 		},
 		{
-			"checksumSHA1": "3tWZA853zC/N1yoGDEkkye76Vv0=",
+			"checksumSHA1": "1ny6UV3Oq4kaqpQtM8bs9W9e3vw=",
```

Comment:
I am afraid that beside this package `github.com/containerd/containerd` we still have some sub packages in vendor.json, such as `github.com/containerd/containerd/api/services/containers/v1`, `github.com/containerd/containerd/api/services/content/v1`.

I have two parts to confirm:

* Do we still need all the sub packages?
* If the above answer is yes, do we need to make all sub packages and containerd package with the same version `v1.0.3`?

Seems to have discussion offline with @rudyfly , please add some comment if you have any thoughts.

Reply:
Good point. I will check the sub-packages.

Reply:
@allencloud and @rudyfly 

I use `govendor fetch github.com/containerd/containerd/...@v1.0.3` to fetch the v1.0.3. But this command will import the `github.com/Microsoft/go-winio` package in the vendor. Since we have the package in the `libnetwork/Godeps`, in order to keep the consistent, I decided to remove the package after update command `govendor fetch github.com/containerd/containerd/...@v1.0.3`. 

If you have any questions, please add the comment. Thanks